src/AddProjectsPage/AddProjectsPage.tsx

```diff
@@ -164,22 +189,38 @@ class AddProjectsPage extends React.Component<
   }
 
   public handleSubmit = (e: React.FormEvent<HTMLButtonElement>): void => {
-    this.props.addProject({
-      name: this.state.name,
-      description: this.state.description,
-      dueDate: this.state.dueDate,
-      team: this.state.team,
-      githubLink: this.state.githubLink,
-      mockupLink: this.state.mockupLink,
-      liveLink: this.state.liveLink,
-      lookingFor: this.state.lookingFor,
-      status: this.state.status,
-      category: this.state.category,
-      tags: this.state.tags,
-      images: this.state.images,
-      contact: this.state.contact,
-      creator: this.props.user.username,
-      files: this.state.files
+    var refToThis = this;
```

Comment:
You don't need to do this if you bind the function in the constructor or use arrow functions. This is not recommended as it is hard to reason about.

You should also update this wherever else you're doing this.